apps/server/src/modules/me/api/me.uc.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { SchoolService } from '@src/modules/school';
+import { UserService } from '@src/modules/user';
+import { EntityId } from '@shared/domain/types';
+import { MeResponseMapper } from './mapper';
+import { MeResponse } from './dto';
+
+@Injectable()
+export class MeUc {
+	constructor(private readonly schoolService: SchoolService, private readonly userService: UserService) {}
+
+	// no authorization is required
+	public async getMe(userId: EntityId, schoolId: EntityId, accountId: EntityId): Promise<MeResponse> {
+		const [school, user] = await Promise.all([
+			this.schoolService.getSchoolById(schoolId),
+			this.userService.getUserEntityWithRoles(userId), // need to replace is valid user domain object avaible
```

Comment:
Ein Todo-Kommentar sollte mMn mit "TODO: " anfangen. Sonst kann man die nicht unterscheiden von anderen Kommentaren und auch schlecht wiederfinden. Ich nehme an, du hast das weggelassen, weil Sonarcloud es dann anmerkt, aber das ist kein guter Grund, finde ich. Dann lieber das Issue in Sonarcloud ignorieren.

Reply:
Also ich kann da ein TODO: hinzufügen. Das stört mich nicht wirklich.

Reply:
hinzugefügt

Reply:
Kann dein Argument auch verstehen. 

Jedenfalls würde ich den Kommentar noch ändern in "// replace when user domain object is available"

Reply:
...oder mit TODO: :)